packages/xo-web/src/xo-app/pool/tab-patches.js

```diff
@@ -197,12 +225,20 @@ export default class TabPatches extends Component {
               {ROLLING_POOL_UPDATES_AVAILABLE && (
                 <TabButton
                   btnStyle='primary'
-                  disabled={isEmpty(missingPatches) || isSingleHost}
+                  disabled={isEmpty(missingPatches) || this.nVmsRunningOnLocalStorage() > 0 || isSingleHost}
```

Comment:
To code duplication: `this.nVmsRunningOnLocalStorage() > 0`, you can directly do a variable that do the check. Like for `isSingleHost`